minotaur.cow

```diff
@@ -0,0 +1,10 @@
+$the_cow = <<"EOC";
+        $thoughts   ^__^
+         $thoughts  ($eyes)
```

Comment:
Sorry I do not understand
```
$ cat .cowsay/minotaur.cow
$the_cow = <<"EOC";
        $thoughts   ^__^
         $thoughts  ($eyes)
            (__)
           /-||-\\
           \\|\\/|/
            o==o
            ||||
            ()()
EOC
$ cowsay -f minotaur You shall not escape my mausoleum!
 ____________________________________
< You shall not escape my mausoleum! >
 ------------------------------------
        \   ^__^
         \  (oo)
            (__)
           /-||-\
           \|\/|/
            o==o
            ||||
            ()()
$ vim .cowsay/minotaur.cow
$ cat .cowsay/minotaur.cow
$the_cow = <<"EOC";
        $thoughts   ^__^
         $thoughts  ($eyes);
            (__)
           /-||-\\
           \\|\\/|/
            o==o
            ||||
            ()()
EOC
$ cowsay -f minotaur You shall not escape my mausoleum!
 ____________________________________
< You shall not escape my mausoleum! >
 ------------------------------------
        \   ^__^
         \  (oo);
            (__)
           /-||-\
           \|\/|/
            o==o
            ||||
            ()()
```

It looks kind of like an ear with an earring, which I guess is kind of cool but not what I was going for.